scipy/stats/_binned_statistic.py

```diff
@@ -12,26 +12,28 @@
 
 
 BinnedStatisticResult = namedtuple('BinnedStatisticResult',
-                                   ('statistic', 'bin_edges', 'binnumber'))
+                                   ('statistic', 'bin_edges', 'binnumbers'))
 
 
 def binned_statistic(x, values, statistic='mean',
                      bins=10, range=None):
     """
-    Compute a binned statistic for a set of data.
+    Compute a binned statistic for one or more sets of data.
 
     This is a generalization of a histogram function.  A histogram divides
     the space into bins, and returns the count of the number of points in
     each bin.  This function allows the computation of the sum, mean, median,
-    or other statistic of the values within each bin.
+    or other statistic of the values (or set of values) within each bin.
 
     Parameters
     ----------
-    x : array_like
+    x : (N,) array_like
         A sequence of values to be binned.
-    values : array_like
-        The values on which the statistic will be computed.  This must be
-        the same shape as `x`.
+    values : (N,) array_like or set of (N,) array_like
```

Comment:
Right, I didn't think about how `set` is a particular type.  At the same time, this could be an `ndarray` instead of a "list" *per se*.  Is "list" still the best terminology?